interop/executor.go

```diff
@@ -126,6 +129,8 @@ func (e *Executor) Execute(ctx context.Context, signedTx tx.SignedTx) error {
 	if err != nil {
 		return fmt.Errorf("failed to get batch from our node, error: %s", err)
 	}
+	log.Debugf("get batch by number: %v", batch)
```

Comment:
Are we ok with logging the entire object, as it has many fields?

Reply:
It's debug log, I'd say this is desired, as much as we can?

Reply:
Ok, my main concern is readability of the logs, even though it is the debug one.